qaframework-bdd-tests/src/test/resources/org/openmrs/uitestframework/test-local.properties

```diff
@@ -1,4 +1,4 @@
-webapp.url=http://localhost:8080/openmrs
+webapp.url=https://qa-refapp.openmrs.org/openmrs
```

Comment:
@sherrif10 the tests are passing github actions after switching to qa instance. So the issue might be coming from the image being used by the local instance.

Reply:
Let me test using assertion texts instead.